spark/v3.3/spark/src/test/java/org/apache/iceberg/spark/source/TestStructuredStreamingRead3.java

```diff
@@ -476,7 +479,15 @@ public void testReadStreamWithSnapshotTypeOverwriteErrorsOut() throws Exception
             dataDeletes,
             deleteRowSchema);
 
-    table.newRowDelta().addDeletes(eqDeletes).commit();
+    DataFile dataFile =
```

Comment:
the test is specifically testing for `OVERWRITE`, hence I'm adding a data file here